docs/conf/eu/2017/index.rst

```diff
@@ -140,6 +140,22 @@
 
                 <div class="row">
                 <div class="col-xs-12 col-md-4">
+                  <a href="news/announcing-schedule/" class="well news__tile">
+                    <h3 class="well__title">
+                      Announcing Presentation Schedule
+                    </h3>
+                    <p class="well__paragraph">
+                    Our website is now updated with the final schedule for
+                    the presentations...
```

Comment:
for some reason presentations rubs me the wrong way. Talks, sessions, topics, anything else! please